src/nvim/tui/tui.c

```diff
@@ -933,12 +986,14 @@ static void out(void *ctx, const char *str, size_t len)
   data->bufpos += len;
 }
 
-static void unibi_set_if_empty(unibi_term *ut, enum unibi_string str,
-    const char *val)
+static bool unibi_set_if_empty(unibi_term *ut, enum unibi_string str,
+                               const char *val)
 {
   if (!unibi_get_str(ut, str)) {
     unibi_set_str(ut, str, val);
+    return true;
   }
+  return false;
```

Comment:
This is rather pointless, since nothing uses the return value; and for what it's worth I have something in the works that I hope will do away with quite a number of calls to this function.